src/profile-logic/gecko-profile-versioning.js

```diff
@@ -331,5 +331,78 @@ const _upgraders = {
     }
     convertToVersionTenRecursive(profile);
   },
+  [11]: profile => {
+    const categories = [
+      {
+        name: 'Idle',
+        color: 'transparent',
+      },
+      {
+        name: 'Other',
+        color: 'grey',
+      },
+      {
+        name: 'JavaScript',
+        color: 'yellow',
+      },
+      {
+        name: 'Layout',
+        color: 'purple',
+      },
+      {
+        name: 'Graphics',
+        color: 'green',
+      },
+      {
+        name: 'DOM',
+        color: 'blue',
+      },
+      {
+        name: 'GC / CC',
+        color: 'orange',
+      },
+      {
+        name: 'Network',
+        color: 'lightblue',
+      },
+      {
+        name: 'HTML Parsing',
+        color: 'brown',
+      },
```

Comment:
This last one category is absent in a profile captured with your gecko patch, is it expected ? Moreover it's not used in your mapping below. So maybe it shouldn't be added at all?
(of course if you remove it here, please remove it also from the other upgrading path)

Reply:
Good catch! I'll remove it.